ClassNotFoundException: com.sun.xml.internal.messaging.saaj.util.ByteOutputStream

Exception while publishing a plugin

```
Caused by: java.lang.NoClassDefFoundError: com/sun/xml/internal/messaging/saaj/util/ByteOutputStream
        at org.jetbrains.intellij.pluginRepository.CompositeConverter.fromBody(pluginRepositoryRest.kt:93)
        at retrofit.RetrofitError.getBodyAs(RetrofitError.java:137)
        at retrofit.RetrofitError.getBody(RetrofitError.java:112)
        at org.jetbrains.intellij.pluginRepository.PluginRepositoryInstance.retrofitErrorMessage(pluginRepositoryRest.kt:219)
        at org.jetbrains.intellij.pluginRepository.PluginRepositoryInstance.processRetofitError(pluginRepositoryRest.kt:202)
        at org.jetbrains.intellij.pluginRepository.PluginRepositoryInstance.uploadPluginInternal(pluginRepositoryRest.kt:167)
        at org.jetbrains.intellij.pluginRepository.PluginRepositoryInstance.uploadPluginInternal$default(pluginRepositoryRest.kt:153)
        at org.jetbrains.intellij.pluginRepository.PluginRepositoryInstance.uploadPlugin(pluginRepositoryRest.kt:138)
        at org.jetbrains.intellij.pluginRepository.PluginRepositoryInstance.uploadPlugin$default(pluginRepositoryRest.kt:137)
        at org.jetbrains.intellij.pluginRepository.PluginRepositoryInstance.uploadPlugin(pluginRepositoryRest.kt)
        at org.jetbrains.intellij.pluginRepository.PluginRepositoryInstance$uploadPlugin.call(Unknown Source)
        at org.jetbrains.intellij.tasks.PublishTask.publishPlugin(PublishTask.groovy:130)
```

This appears to have been introduced in https://github.com/JetBrains/plugin-repository-rest-client/commit/c132ac89713a134edcb37f706c1200e008b3d99d. I think this is an import mistake: I assume`import com.sun.xml.internal.messaging.saaj.util.ByteOutputStream` should be `import java.io. ByteArrayOutputStream`

--- a/rest/src/main/kotlin/org/jetbrains/intellij/pluginRepository/pluginRepositoryRest.kt
+++ b/rest/src/main/kotlin/org/jetbrains/intellij/pluginRepository/pluginRepositoryRest.kt
@@ -1,7 +1,6 @@
 package org.jetbrains.intellij.pluginRepository
 
 import com.google.gson.Gson
-import com.sun.xml.internal.messaging.saaj.util.ByteOutputStream
 import org.jetbrains.intellij.pluginRepository.exceptions.UploadFailedException
 import org.simpleframework.xml.Attribute
 import org.simpleframework.xml.Element
@@ -21,6 +20,7 @@ import retrofit.mime.TypedFile
 import retrofit.mime.TypedInput
 import retrofit.mime.TypedOutput
 import retrofit.mime.TypedString
+import java.io.ByteArrayOutputStream
 import java.io.File
 import java.io.IOException
 import java.lang.reflect.Type
@@ -90,7 +90,7 @@ class CompositeConverter : Converter {
             jsonConverter.fromBody(body, type)
         }
         else -> {
-            val outputStream = ByteOutputStream()
+            val outputStream = ByteArrayOutputStream()
             body.`in`().copyTo(outputStream)
             outputStream.toString()
         }

